### ISSUE TITLE
A11y Bug 9948579: [OneNote Clipper] [Web]: Keyboard focus is not moving to default location dropdown with tab key.

### ISSUE THREAD

**Issue by KethanaReddy7** [OneNoteDev/WebClipper#631]
Environment Details:
Application Name: Excel Start - In Grid Create with Copilot.
Platform: Web
Browser version: Version 135.0.3179.33 (Official build) beta (64-bit)
Application URL: https://www.onenote.com/clipper
Repro Steps:
Open https://www.onenote.com/clipper.
Navigate till get one note web clipper for edge and activate it.
Edge Add-ons page will appear. Invoke the Get button.
OneNote Clipper will be added in edge extension section. 
Expand the Extension button and turn on the OneNote Clipper Show in Toolbar button.
Now invoke the OneNote clipper button.
Sign in with V-id.
Try to navigate to location dropdown with tab key.
Observe the issue.
Actual Result:
Keyboard focus is not moving to default location dropdown with tab key. Only by using shift+ tab key we can be able to move to default location dropdown.
Expected Result:
Keyboard focus should be able to default location dropdown by using tab key.
User Impact:
Users will face difficulty if keyboard focus is not moving to default location dropdown with tab key.
MAS Reference:
[MAS 2.4.3 – Focus Order](https://aka.ms/MAS2.4.3)
Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.

**Comment by KethanaReddy7** [comment]
Issue fixed in the PR https://github.com/OneNoteDev/WebClipper/pull/632
